process/track/baseBlockTrack_test.go

```diff
@@ -631,6 +710,27 @@ func TestAddHeader_ShouldNotAddIfItAlreadyExist(t *testing.T) {
 	assert.Equal(t, header, headers[0])
 }
 
+func TestAddHeader_ShouldCleanupWhenMaxCapacityIsReached(t *testing.T) {
+	t.Parallel()
+	shardArguments := CreateShardTrackerMockArguments()
+	sbt, _ := track.NewShardBlockTrack(shardArguments)
+
+	maxNumHeadersToKeepPerShard := 1200
```

Comment:
please define functions in export_test.go where you export the 2 constants because you hardcoded them in each test you wrote in this PR and if we change the constants we would have to adapt all the tests.

Reply:
Done